src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/Create/ClassRoomCreateForm.spec.tsx

```diff
@@ -51,8 +53,8 @@ describe('<ClassRoomCreateForm />', () => {
     ).toBeInTheDocument();
   });
 
-  it('field mandatory', async () => {
-    render(<ClassRoomCreateForm onSubmit={jest.fn()} />);
+  test('fields mandatory', async () => {
```

Comment:
FMI, what's the difference between `test` and `it` ?

Reply:
I think it is exactly the same. I changed it because `test` was used in the other tests.

Reply:
`it` is an alias of `test`. `it` allows to construct a sentence.